packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -46,6 +46,8 @@ describe('validation', function() {
         errors
       } = validateTemplate(template);
 
+      console.log(JSON.stringify(errors, null, 2));
```

Comment:
For this one, the test files provide a [friendly utility](https://github.com/camunda/element-templates-json-schema/blob/2610dd08b4c8360597c42ad9fb1a5c0c3b1e4e55/packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js#L202). Pretty handy 🙂

Reply:
Uh, nice one!